docs/e2e.md

```diff
@@ -0,0 +1,152 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# End to End
+
+RamenDR end-to-end (e2e) tests validate various scenarios for regional DR
+using predefined workloads and deployment methods.
+
+## Prepare `config.yaml` File
+
+Create a `config.yaml` file by copying the `config.yaml.sample` template:
+
+```sh
+cat e2e/config.yaml.sample >> e2e/config.yaml
+```
+
+Update `config.yaml` by adding kubeconfig paths of the clusters, with format:
+
+```yaml
+Clusters:
+    c1:
+        kubeconfigpath: /path/to/kubeconfig/c1
+    c2:
+        kubeconfigpath: /path/to/kubeconfig/c2
+    hub:
+        kubeconfigpath: /path/to/kubeconfig/hub
+```
+
+If `drenv` was used to configure RDR clusters, easily copy `config.yaml.sample`
+and add the clusters kubeconfig paths into `config.yaml` using:
+
+```sh
+cat e2e/config.yaml.sample ~/.config/drenv/rdr/config.yaml >> e2e/config.yaml
+```
+
+## Run All E2E Tests
+
+`run.sh` is a shell script that runs the RamenDR E2E tests for regional DR.
+
+To run all the tests:
+
+```sh
+cd e2e
+./run.sh
+```
+
+## Run Tests with Custom Config File
+
+Use this option if you want to maintain multiple configuration files and run
+tests using a specific one. Example usage:
+
+```sh
+./run.sh -config my_config.yaml
+```
+
+## Run Specific E2E Tests
+
+The `-run` option is commonly used when debugging a failing test, developing
+a new test, or working on a new deployer or workload. It allows to selectively
+execute specific tests by matching full test names using regular expressions,
+making it easier to focus on specific scenarios.
+
+Refer to the **List of Tests** section below for a detailed breakdown of
+available tests.
```

Comment:
We can create link to another section using this format:

```
[List of tests](#list-of-tests)
```

See https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#custom-anchors